### ISSUE TITLE
Bad request on Disk usage page

### ISSUE THREAD

**Issue by DavidePrincipi** [NethServer/dev#5359]
The Disk usage page fails to update the usage graph.

**Steps to reproduce**

- Go to page "Disk usage"
- Push "Update"

**Expected behavior**

Disk usage recalculate procedure starts

**Actual behavior**

The error dialog appears:

```text
Nethgui:

    400 - Bad request

    1504102184+1504102187
```

In /var/log/messages

```text
Oct 13 11:54:57 vm5 httpd: [ERROR] Nethgui\Framework: CSRF token verification failed!
```

**Components**

nethserver-duc-1.4.2-1.ns7.noarch

----

Thanks to @gsanchietti for pointing it out

**Comment by gsanchietti** [comment]
In 7.4/nethserver-testing:
nethserver-duc-1.4.2-1.2.ga5a1112.ns7.noarch.rpm

**Test case**
- Check the bug is not reproducible

**Comment by gsanchietti** [comment]
Verified.

**Comment by gsanchietti** [comment]
Released in 7.4/nethserver-updates:
nethserver-duc-1.4.3-1.ns7.noarch.rpm

**Comment by gsanchietti** [comment]
Released also for ns NS 6.9:
nethserver-duc-1.0.7-1.ns7.noarch.rpm
